org.eclipse.scout.rt.platform.test/src/test/java/org/eclipse/scout/rt/platform/config/PropertiesHelperTest.java

```diff
@@ -339,14 +303,14 @@ public void testReadPropertyMapFromEnvironmentWithVariableReference() {
           "\"propFromConfigPropertiesInString\": \"test${longKey}testtest\"," +
           "\"propFromEnvFileInString\": \"test${envFileKey}testtest\"}";
 
-      BEANS.get(TestJsonPropertyReader.class).putMapping(mapNotInFilePropertyValue, CollectionUtility.hashMap(
-          new ImmutablePair<>("propFromConfigProperties", "${otherProp}"),
-          new ImmutablePair<>("propFromSystemProperties", "${sysProp}"),
-          new ImmutablePair<>("propFromEnvironment", "${envProp}"),
-          new ImmutablePair<>("propFromConfigPropertiesOverriddenByEnv", "${intKey}"),
-          new ImmutablePair<>("propFromConfigPropertiesOverriddenBySystemProperty", "${stringKey}"),
-          new ImmutablePair<>("propFromConfigPropertiesInString", "test${longKey}testtest"),
-          new ImmutablePair<>("propFromEnvFileInString", "test${envFileKey}testtest")));
+      //      BEANS.get(TestJsonPropertyReader.class).putMapping(mapNotInFilePropertyValue, CollectionUtility.hashMap(
```

Comment:
Do you want to comment out those lines?

Reply:
No, of course not; thanks for the hint - change is not yet ready for review - awaiting more changes/another force-push.